src/flask_rq2/cli.py

```diff
@@ -118,8 +118,8 @@ def info(rq, ctx, path, interval, raw, only_queues, only_workers, by_queue,
 @click.option('--name', '-n', help='Specify a different name')
 @click.option('--path', '-P', default='.', help='Specify the import path.')
 @click.option('--results-ttl', help='Default results timeout to be used')
```

Comment:
It would be nice to also update this option with `type=int` and maybe a default. 

Been noticing this error when providing a value or a similar error (instead of str complains about None) without providing a value.
```Traceback (most recent call last):
  File \"/usr/lib/python3.6/site-packages/rq/worker.py\", line 803, in perform_job
    started_job_registry=started_job_registry)
  File \"/usr/lib/python3.6/site-packages/rq/worker.py\", line 766, in handle_job_success
    finished_job_registry.add(job, result_ttl, pipeline)
  File \"/usr/lib/python3.6/site-packages/rq/registry.py\", line 43, in add
    score = ttl if ttl < 0 else current_timestamp() + ttl
TypeError: '<' not supported between instances of 'str' and 'int'
", "exc_info": "Traceback (most recent call last):
  File \"/usr/lib/python3.6/site-packages/rq/worker.py\", line 803, in perform_job
    started_job_registry=started_job_registry)
  File \"/usr/lib/python3.6/site-packages/rq/worker.py\", line 766, in handle_job_success
    finished_job_registry.add(job, result_ttl, pipeline)
  File \"/usr/lib/python3.6/site-packages/rq/registry.py\", line 43, in add
    score = ttl if ttl < 0 else current_timestamp() + ttl
TypeError: '<' not supported between instances of 'str' and 'int'
```